harness/determined/tensorboard/metric_writers/pytorch.py

```diff
@@ -1,6 +1,6 @@
-from typing import Any, Union
-
-import numpy as np
+# mypy: ignore-errors
```

Comment:
If you need a module-level comment like this, you will need a comment explaining why it's necessary, otherwise it looks very sketch